apps/vx-scan/frontend/src/utils/use_sound.ts

```diff
@@ -0,0 +1,6 @@
+import useSoundLib from 'use-sound';
+
+export function useSound(sound: 'success' | 'warning' | 'error'): () => void {
+  const [playSound] = useSoundLib(`/sounds/${sound}.mp3`);
+  return playSound;
+}
```

Comment:
This file got moved. I also moved the mute logic out of this hook into the voter screen where the sounds get played